Task2_3/Training.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.IO;
+using System.Security.Cryptography;
+
+namespace Task2_3
+{
+    internal class Training
+    {
+        public string? Description { get; set; }

```

Comment:
Create base class Lectures and Practical Lessons, something like Lesson, and then:
 public Lesson[]? LecturesAndLessons { get; set; }

Reply:
Request Completed. Waiting for review